config/sync/core.entity_form_display.node.article.default.yml

```diff
@@ -54,6 +54,16 @@ content:
       progress_indicator: throbber
       preview_image_style: thumbnail
     third_party_settings: {  }
+  field_ingredient_terms:
```

Comment:
Although we used this field for testing during the standup, in the end this field should be hidden, so please hide it again. :)

Reply:
Fixed!